src/data/pages/users.ts

```diff
@@ -0,0 +1,190 @@
+  },
+  {
+    id: 3,
+    fullname: 'Liz Macintosh',
+    email: 'ebrown@gmail.com',
+    username: 'ebrown',
+    role: 'user',
+    projects: 2,
+    avatar: 'https://randomuser.me/api/portraits/men/1.jpg',
+    active: true,
+    notes: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, voluptatum.',
+  },
+  {
+    id: 4,
+    fullname: 'M2',
+    email: 'mrm@gmail.com',
+    username: 'mrm',
+    role: 'owner',
+    projects: 1,
+    avatar: '',
+    active: true,
+    notes: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, voluptatum.',
+  },
+  {
+    id: 5,
+    fullname: 'Kevin Smith',
+    email: 'kevin@gmail.com',
+    username: 'kevin13',
+    role: 'user',
+    projects: 13,
+    avatar: 'https://randomuser.me/api/portraits/men/2.jpg',
+    active: true,
+    notes: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, voluptatum.',
+  },
+  {
+    id: 6,
+    fullname: 'Martin Hoff',
+    email: 'martin@gmail.com',
+    username: 'martin3',
+    role: 'user',
+    projects: 2,
+    avatar: 'https://randomuser.me/api/portraits/men/3.jpg',
+    active: true,
+    notes: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, voluptatum.',
+  },
+  {
+    id: 7,
+    fullname: 'John Doe',
+    email: 'john@mail.com',
+    username: 'john',
+    role: 'user',
+    projects: 1,
+    avatar: '',
+    active: true,
+    notes: '',
+  },
+  {
+    id: 8,
+    fullname: 'Maksim Nedo',
+    email: 'maksim@epic.com',
+    username: 'maksim',
+    role: 'admin',
+    projects: 1,
+    avatar: 'https://avatars.githubusercontent.com/u/23530004?v=4',
+    active: true,
+    notes: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, voluptatum.',
+  },
+  {
+    id: 9,
+    fullname: 'Dmitry Kuzmenko',
+    email: 'dd@pp.com',
+    username: 'dd',
+    role: 'user',
+    projects: 1,
+    avatar: '',
+    active: true,
+    notes: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, voluptatum.',
+  },
+  {
+    id: 10,
+    fullname: 'Rayan Gosling',
+    email: 'rayan@u.ua',
+    username: 'rayan',
+    role: 'user',
+    projects: 1,
+    avatar: '',
+    active: true,
+    notes: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, voluptatum.',
+  },
+  {
+    id: 11,
+    fullname: 'Laura Smith',
+    email: 'laura@example.gg',
+    username: 'bbb',
+    role: 'user',
+    projects: 3,
+    avatar: '',
+    active: false,
+    notes: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, voluptatum.',
+  },
+  {
+    id: 12,
+    fullname: 'Ted Mosby',
+    email: 'tedmosby@mail.com',
+    username: 'gamer777',
+    role: 'user',
+    projects: 5,
+    avatar: '😭',
+    active: false,
+    notes: 'Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, voluptatum.',
+  },
+]).map((u, index) => ({ ...u, id: index }))
+
+// Simulate API calls
+export type Filters = {
+  isActive: boolean
```

Comment:
Could you check what's wrong with checkbox? Height is less than width. Should be the same.

![image](https://github.com/epicmaxco/vuestic-admin/assets/5394573/dd82b407-8aef-46c5-b9e2-d082f24dbe0a)

Reply:
Looks OK for me. Let's have issue in VuesticUI for that. 
<img width="90" alt="image" src="https://github.com/epicmaxco/vuestic-admin/assets/23530004/3320f8bd-76d1-476f-9bb0-34d074f7794d">

Reply:
Hmm maybe something to do with rounding of 17.5. I.e. vertically it goes 17, horizontally 18.

But yeah, pixel hunting...